index.d.ts

```diff
@@ -410,6 +533,25 @@ declare module 'warframe-items' {
         levelStats: LevelStat[];
     }
 
+    interface TitaniaRelicLocation {
+        location: string;
+        rarity: Rarity;
+        chance: number;
+    }
+    interface TitaniaRelicReward {
+        rarity: Rarity;
+        chance: number;
+        item: TitaniaRelicRewardItem;
+    }
+    interface TitaniaRelicRewardItem {
```

Comment:
Should it extends `MinimalItem`, by having an unique name? If so i think it would be `warframe-items` responsibility to add it. 
```suggestion
    interface TitaniaRelicRewardItem extends MinimalItem {
```

Reply:
It doesn't have a uniqueName, I don't think...

Reply:
> It doesn't have a uniqueName, I don't think...

Yeah that's why it would be `warframe-items` responsibility to add it. By checking the for an item with the same name and getting it's uniqueName, and probably we would have to make some other guesses to narrow it down.

Reply:
Ah, I see what you mean. We could do that.

Reply:
Should it be included in this PR or in another one?

Reply:
Before I merge this, I'm doing some ci work on relics, so we can probable do that on there shortly. Ideally we'll be able to drop our extra types

Reply:
looks like we can't drop them just yet, and we don't have unique names on them (yet)... do we want that before initial support?

Reply:
> looks like we can't drop them just yet, and we don't have unique names on them (yet)... do we want that before initial support?

No need, it's just a nice to have in case we want to find more information about some item.

Reply:
unfortunately rebasing has some ci issues, so i'll get those sorted, probably rebase again, then poke again for re-review